protos/sources/vega/vega.proto

```diff
@@ -1621,3 +1621,12 @@ message BenefitTier {
   // Proportion of the referee's taker fees to be discounted
   string referral_discount_factor = 4;
 }
+
+message VestingBenefitTiers {
+  repeated VestingBenefitTier tiers = 1;
+}
+
+message VestingBenefitTier {
+  string minimum_quantum_balance = 1;
+  string reward_multiplier = 2;
```

Comment:
Please add doc strings here

Reply:
this is never used in an API really, it's uncessary...

Reply:
Is it not used when the community want to update the tiers?

Reply:
not really, only internally as network parameters are all just strings